### ISSUE TITLE
`scala-cli --jvm adopt:11 Main.scala` fails

### ISSUE THREAD

**Issue by carueda** [VirtusLab/scala-cli#354]
Repro:

Starting from `brew install Virtuslab/scala-cli/scala-cli`, then running the `Files.writeString ` example at <https://scala-cli.virtuslab.org/docs/cookbooks/scala-jvm>:

```
$ scala-cli version
0.0.7

$ scala-cli --jvm adopt:11 Main.scala

Compiling project (Scala 3.0.2, JVM)
[error] ./Main.scala:6:18: value writeString is not a member of object java.nio.file.Files
[error]   val filePath = Files.writeString(dest, "Hello from ScalaCli")
[error]                  ^^^^^^^^^^^^^^^^^
[error] ./Main.scala:7:29: value readString is not a member of object java.nio.file.Files
[error]   val fileContent: String = Files.readString(filePath)
[error]                             ^^^^^^^^^^^^^^^^
Error compiling project (Scala 3.0.2, JVM)
Compilation failed
```

More info:

Using java via `sdk`.

```
$ java -version
openjdk version "17.0.1" 2021-10-19
OpenJDK Runtime Environment Temurin-17.0.1+12 (build 17.0.1+12)
OpenJDK 64-Bit Server VM Temurin-17.0.1+12 (build 17.0.1+12, mixed mode, sharing)
```

(btw, amazing project!)

**Comment by romanowski** [comment]
Hi @carueda, thank you for reporting and we are really happy that you are enjoying Scala CLI.

I suspect that the problem was caused by the wrong version of JVM used by Bloop (the build server used under the hood) to compile your code.

@tpasternak fixed the problem in https://github.com/VirtusLab/scala-cli/pull/217 and it was included in the latest release (`0.0.8`). 

In case the problem is not fixed after the update, please let us know and we will open this issue and investigate.

**Comment by carueda** [comment]
perfect!  (verified ok)
